examples/problem_decomposition/ifci_mifno.ipynb

```diff
@@ -0,0 +1,554 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "(All the energy values are in hartree)\n",
+      "Total incremental energy = -15.83647358459995\n",
+      "Correlation energy = -0.06915435696141081\n",
+      "Mean-field energy = -15.76731922763854\n",
+      "        energy_total  energy_correlation   epsilon  correction  n_electrons  \\\n",
+      "(0,)      -15.767649           -0.000329 -0.000329   -0.000329            2   \n",
+      "(1,)      -15.782341           -0.015022 -0.015022   -0.000553            2   \n",
+      "(2,)      -15.784406           -0.017087 -0.017087   -0.000844            2   \n",
+      "(0, 1)    -15.782980           -0.015661 -0.000309   -0.000850            4   \n",
+      "(0, 2)    -15.785090           -0.017771 -0.000354   -0.001301            4   \n",
+      "(1, 2)    -15.835480           -0.068161 -0.036052   -0.000222            4   \n",
+      "\n",
+      "        n_spinorbitals  \n",
+      "(0,)                 2  \n",
+      "(1,)                26  \n",
+      "(2,)                18  \n",
+      "(0, 1)              28  \n",
+      "(0, 2)              20  \n",
+      "(1, 2)              38  \n"
+     ]
+    }
+   ],
+   "source": [
+    "for file in os.listdir(data_folder):\n",
+    "    if file.endswith(\".log\"):\n",
+    "        log_file_name = os.path.join(data_folder, file)\n",
+    "\n",
+    "fno_fragments = MethodOfIncrementsHelper(log_file=log_file_name)\n",
+    "print(fno_fragments)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "It parsed information contained in the \"result blob\" from QEMIST Cloud. We see that there is information about the whole system, including the mean-field energy and the total energy, as well as how each of the fragment -denoted by a tuple of integers- contributed to the correlation energy. Here are the label breakdown:\n",
+    "- *energy_total*: Total energy of the fragment, with the correction term.\n",
+    "- *energy_correlation*: Total energy minus the mean-field energy.\n",
+    "- *epsilon*: Contribution to the MI summation.\n",
+    "- *correction*: MP2 correction, coming from the truncation of the virtual space in MI-FNO.\n",
+    "- *n_electrons*: Number of active electrons in the fragment.\n",
+    "- *n_spinorbitals*: Number of active spinorbitals in the fragment.\n",
+    "\n",
+    "Although not displayed here, the object also contains information about the frozen orbitals used for each of these fragments (\"frozen lists\")."
```

Comment:
Maybe we could mention that the number of electrons and spin orbitals give us a sense of how much resources each subproblem requires in order to be resolved. (just as a "at a glance" observation)